renv.lock

```diff
@@ -686,6 +826,13 @@
       "Repository": "CRAN",
       "Hash": "5ba3006888ac62fd5e97b208d00d3317"
     },
+    "viridis": {
+      "Package": "viridis",
+      "Version": "0.6.1",
+      "Source": "Repository",
+      "Repository": "CRAN",
+      "Hash": "424d1285b7372d39806abb19467e4df4"
+    },
     "viridisLite": {
       "Package": "viridisLite",
       "Version": "0.3.0",
```

Comment:
The [travis.ci pipeline](https://travis-ci.org/github/Hong-Kong-Districts-Info/hktrafficcollisions/builds/771319974) fails because it says that this version of the `viridis` package requires the `viridislite` package to be above 0.4.0.
```shell script
Error: package ‘viridisLite’ 0.3.0 was found, but >= 0.4.0 is required by ‘viridis’
```

Reply:
hey @avisionh, thanks for the review. Just to clarify/confirm the steps for upgrading this package (for my own benefit and @KHwong12 as well), is it: 
1. Ensure R Session is loaded with `viridislite` above 0.4.0. 
2. Run `renv::snapshot()` to capture the loaded package version information 
3. Commit - and other users will run `renv::restore()` so that they can run the code from the same versions of package dependencies. 

..?

Reply:
I see. Never used **renv** before and I directly jumped to develop the interactive map. Probably that's why `renv.lock` does not match. Thanks for the guidelines!

Reply:
The `renv` package is super useful for managing package versions, and ensuring code and its outputs are reproducible on other people's machines - highly recommend using it. 👍 

I've see you've updated the version in `renv.lock` already so including this for completeness (in answer of @martinctc's question). It's pretty much what you said @martinctc but with a tweak to Step 1. as below:
1. Run `renv::update(packages=c('viridislite'))`